x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/latency/service_maps.spec.ts

```diff
@@ -73,7 +73,14 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
   let latencyMetricValues: Awaited<ReturnType<typeof getLatencyValues>>;
   let latencyTransactionValues: Awaited<ReturnType<typeof getLatencyValues>>;
-  registry.when('Service Maps APIs', { config: 'trial', archives: [] }, () => {
+
+  describe('Service Maps APIs', () => {
+    let apmSynthtraceEsClient: ApmSynthtraceEsClient;
+
+    before(async () => {
+      apmSynthtraceEsClient = await synthtrace.createApmSynthtraceEsClient();
+    });
```

Comment:
Do you need an after call to clean up, [like this](https://github.com/elastic/kibana/pull/199775/files#diff-e5441e687eb0a324ba74caadbc4cfe82167f81c33484dd66f42c7ac96f7daccfR56)?

Reply:
Yes, and its there https://github.com/elastic/kibana/blob/2914a6ea608492152be4305851bcfd68351f65a5/x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/latency/service_maps.spec.ts#L122